README.md

```diff
@@ -6,4 +6,9 @@ just need to get onto one host in a group, and don't care which.
 Usage:
 ```
 rah --inventory PATH_TO_INVENTORY_FILE_OR_DIRECTORY ssh ANSIBLE_GROUP
-```
\ No newline at end of file
+```
+
+Installation:
+```
+pip install https://github.com/simonft/random-ansible-host/zipball/master
```

Comment:
Any reason to use this over 

`pip install git+https://github.com/simonft/random-ansible-host.git`?